android/build.gradle

```diff
@@ -30,6 +31,6 @@ repositories {
 }
 
 dependencies {
-    compile "com.facebook.react:react-native:+"
-    compile files('libs/zip4j_1.3.2.jar')
+    implementation "com.facebook.react:react-native:+"
+    implementation files('libs/zip4j_1.3.2.jar')
```

Comment:
fwiw my installation of RNZA 3.0.1 does not include this `compile|implementation files(...)` line at all, and if i include it my project doesn't build.

this may be stripped during distribution to NPM, or something similar where for some reason it does not appear on my local, so it may not be a real problem--I just thought I should report it.

otherwise, this PR fixes building on 0.58.4. my root project SDK targets are:

```
buildToolsVersion = "28.0.2"
minSdkVersion = 21
compileSdkVersion = 28
targetSdkVersion = 27
supportLibVersion = "28.0.0"
```

Reply:
note about the above: i realized this is because the shipped tag `3.0.1` is not current with `master`, and 3.0.1 doesn't have the `libs/zip4j` line. so please ignore the bulk of my comment above. :)

Reply:
Thanks for reporting, would have a close look on this.

Reply:
I just tried this on the example app including in this repo but got no errors, could you confirm it again? use the code from the master branch. If there are no more errors, I would release it ASAP.